spec/run.sh

```diff
@@ -48,12 +53,38 @@ setup_certora() {
     echo "Found CERTORAKEY environment variable." >&2
```

Comment:
Just letting you know that attempt to read unset CERTORAKEY fails with `spec/run.sh: line 49: CERTORAKEY: unbound variable` (I believe it has something to do with `set -eu`). Not that it matters since we do want to terminate if CERTORAKEY is empty.

Reply:
Yep! It's not the most helpful error message, but this was intended behavior to make sure we fail early before sending a spec to Certora when we don't have a CERTORAKEY.

@skalermo I've put in a request to grant you an API key.